test/tokenize.jl

```diff
@@ -44,12 +44,13 @@ end
 end # testset
 
 @testset "tokenize unicode" begin
-    str = "𝘋 =2β"
+    emoji = VERSION < v"1.5" ? "😄" : "🏳️‍🌈" # requires newer Unicode
```

Comment:
Yes, that's right — it's because `is_identifier_char` uses a version of utf8proc (and hence Unicode character classification) that is tied to the Julia version. 

If you load a utf8proc JLL explicitly and implement your own `is_identifier_char`, then your Unicode version will become Julia-version independent.

Reply:
```suggestion
    # FIXME: rm VERSION check once we implement our own is_identifier_char
    emoji = VERSION < v"1.5" ? "😄" : "\U1F3F3\UFE0F\U200D\U1F308" # 🏳️‍🌈 requires newer Unicode
```